docs/uuid.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Basic JSON Editor Example</title>
+    <script src="https://cdn.jsdelivr.net/npm/@json-editor/json-editor@latest/dist/jsoneditor.min.js"></script>
+  </head>
+  <body>
+    <h1>UUID JSON Editor Example</h1>
+    
+    <div id='editor_holder'></div>
+    <button id='submit'>Submit (console.log)</button>
+
+    <script>
+      JSONEditor.defaults.options['required_by_default'] = true;
+      // Initialize the editor with a JSON schema
+      var editor = new JSONEditor(document.getElementById('editor_holder'),{
+        schema: {
+          "type": "object",
+          "title": "UUID format with autogenerated uuid",
+          "properties": {
+            "_header": {
+              "type": "string",
+              "format": "hidden",
+              "description": "Example of new editor format [uuid] for string type. Value must be a valid RFC 4122 uuid (Universally Unique IDentifier). If field has initial no value (startval) then a random uuid will be autogenerated",
+              "options": {
+                "compact": true
+              }
+            },
+            "uuid1": {
+              "type": "string",
+              "format": "uuid",
```

Comment:
How about changing this into `"format": "uuid-v4",` or eg. adding something like *options*. There could also be others versions of UUIDs used.

Reply:
Only v4 is randomly generated. The other versions are generated by timestamp or namespace.
https://en.wikipedia.org/wiki/Universally_unique_identifier

If you supply the uuid from other source (i.e. part of startval), then it will validate the id for all uuid versions, as the regex pattern is global.